rezz.lic

```diff
@@ -1,118 +1,131 @@
-#   Documentation: https://elanthipedia.play.net/Lich_script_repository#rezz
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#rezz
+  
+  Define a waggle sets for rezz and rejuv if you don't want to use the default values in base-spells.yaml
+  
+  EXAMPLE
+  waggle_sets:
+    rezz:
+      Resurrection:
+        mana: 17
+    rejuv:
+      Rejuvenation:
+        mana: 10
+        cambrinth:
+        - 15
+
+  In order to infuse directly from your mana pool add the following to your yaml:
+  osrel_no_harness: true
+  
+=end
 
 custom_require.call(%w[common common-arcana events drinfomon spellmonitor])
 
-# rezz waggle set with POM if you want
-
 class Rezz
   include DRC
   include DRCA
-
+  
   def initialize
+    DRStats.cleric? ? nil : begin echo '*** Nice try, heathen! ***'; exit end
+	
     arg_definitions = [
       [
         { name: 'person', regex: /\w+/, description: 'Person to rezz' }
       ]
     ]
-
-    @rejuv_mana = 0
+	
     args = parse_args(arg_definitions)
     @settings = get_settings
-    rezz_prep(args.person)
+    
+    @person = args.person.capitalize
+	/the body of (#{@person}.*) who/ =~ bput('look people', /the body of (#{@person}.*) who/)
+	@person = Regexp.last_match(1)
+    
+    @rezz = if @settings.waggle_sets['rezz']
+               @settings.waggle_sets['rezz'].values.first
+           else
+	         get_data('spells').spell_data['Resurrection']
```

Comment:
Do we want to print something to the user here? Such as "Using minimum prep values. Please make a waggle set called 'rezz' to configure your rezz spell settings."

Reply:
I think yelling at the user to define their spell settings should be good enough. Doing it at min prep is likely to be a horrible experience and should motivate them :D